test/test_connectionpool.py

```diff
@@ -140,6 +160,33 @@ def test_not_same_host_no_port_https(self, a, b):
         with HTTPSConnectionPool(b) as c:
             assert not c.is_same_host(a)
 
+    @pytest.mark.parametrize('a, b', [
+        ('%2Fvar%2Frun%2Fdocker.sock',
+         'http+unix://%2Fvar%2Frun%2Fdocker.sock'),
+        ('%2Fvar%2Frun%2Fdocker.sock',
+         'http+unix://%2Fvar%2Frun%2Fdocker.sock/'),
+        ('%2Fvar%2Frun%2Fdocker.sock',
+         'http+unix://%2Fvar%2Frun%2Fdocker.sock/abracadabra'),
+        ('%2Ftmp%2FTEST.sock', 'http+unix://%2Ftmp%2FTEST.sock'),
+        ('%2Ftmp%2FTEST.sock', 'http+unix://%2Ftmp%2FTEST.sock/'),
+        ('%2Ftmp%2FTEST.sock', 'http+unix://%2Ftmp%2FTEST.sock/abracadabra'),
+    ])
+    def test_same_host_custom_protocol(self, a, b):
+        with HTTPUnixConnectionPool(a) as c:
+            assert c.is_same_host(b)
+
+    @pytest.mark.parametrize('a, b', [
+        # different case
```

Comment:
You can remove the comments within the parametrize

Reply:
I wanted to make it more clear what is a purpose of rest.
Comments were removed.